src/dtos/dao.dto.ts

```diff
@@ -1,16 +1,53 @@
-export interface NewDaoDto {
-  id?: string;
+import {
+  ArrayNotEmpty,
+  IsArray,
+  IsOptional,
+  IsString,
+  Length,
+  MaxLength,
+} from 'class-validator';
+
+export class CreateDaoDto {
+  @IsString()
+  @Length(1, 50)
```

Comment:
length should be 3-50
```suggestion
  @Length(3, 50)
```